cirq-google/cirq_google/serialization/circuit_serializer.py

```diff
@@ -175,7 +175,9 @@ def _serialize_gate_op(
                 out=msg.phasedxpowgate.exponent,
                 arg_function_language=arg_function_language,
             )
-        elif isinstance(gate, cirq.PhasedXZGate):
+        elif isinstance(gate, (cirq.PhasedXZGate, cirq.ops.SingleQubitCliffordGate)):
```

Comment:
Should this be done explicitly, e.g. with [`optimize_for_target_gateset`](https://github.com/quantumlib/Cirq/blob/7da7f64c712cd2665d842060ec9b3a7525bf73b9/cirq-core/cirq/transformers/optimize_for_target_gateset.py#L99) rather than implicitly during serialization? IIRC, we prefer to make transformations like this explicit to the user.